butterfly-slack/src/main/java/com/paypal/butterfly/slack/SlackNotifierConfig.java

```diff
@@ -0,0 +1,140 @@
+package com.paypal.butterfly.slack;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.paypal.butterfly.api.TransformationListener;
+import com.paypal.butterfly.api.TransformationRequest;
+import com.paypal.butterfly.api.TransformationResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+import javax.annotation.PostConstruct;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.File;
+
+/**
+ * Observer that receives data from Pupa and sends it to
```

Comment:
Please, remove the term `Pupa` from here. That is a PayPal specific system, which doest not exist in the open source world. You can rephrase this to:

```
Observer that notifies the transformation requester via Slack when a transformation finishes.
```